index.html

```diff
@@ -8,50 +8,266 @@
+                    </h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">CANBOY</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Back End Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about ">
+        <div class="card-items">
+            <div class="card-continent">
+                <h1>About <br/> Myself</h1>
+                <p class="about-text">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <p class="contact">LET’S CONNECT</p>
+                <div>
+                    <ul class="about-icons list-style">
+                        <li class="icon ">
+                            <a href="https://twitter.com/ibr_ahmat " target="”_blank”" rel="noreferrer noopener "><img src="./assets/twitter.svg " alt="Twitter icon " /></a>
+                        </li>
+                        <li class="icon ">
+                            <a href="https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/ " target="”_blank”" rel="noreferrer noopener "><img src="./assets/linkedin.svg " alt="Linked-In icon " /></a>
+                        </li>
+                        <li class="icon ">
+                            <a href="# " rel="noreferrer "><img src="./assets/medium.svg " alt="Medium icon " /></a>
+                        </li>
+                        <li class="icon ">
+                            <a href="https://github.com/ibr5500 " target="”_blank”" rel="noreferrer noopener "><img src="./assets/github.svg " alt="Github icon " /></a>
+                        </li>
+                        <li class="icon ">
+                            <a href="# " rel="noreferrer "><img src="./assets/angellist.svg " alt="Angellist icon " /></a>
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <div class="about-button-container button-container ">
+                        <p class="button ">Get My Resume</p>
```

Comment:
- [ ] This element is meant to be a button. Please change the `<p>` tag to a link `<a>` or `<button>` tag, and follow the button styling: Active, Hover, and Pressed.
![Capture](https://user-images.githubusercontent.com/85095702/159127556-1db3c7bd-5d8a-4a13-8f7a-e80366acad3a.PNG)